subgraph.yaml

```diff
@@ -15,7 +15,11 @@ dataSources:
       language: wasm/assemblyscript
       entities:
         - Domain
+        - DomainTransferred
```

Comment:
The event entities don't need to be added here from my understanding

Reply:
I just added all of the entities from the schema.graphql. What differentiates an entity and an event entity?

Reply:
No idea, but I know that ENS doesn't have them listed as entities, I think because maybe these are for top level entities only?